.github/scripts/generate-artifacts.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+repo_root_dir='.'
+deployment_output_folder="$repo_root_dir/broadcast/Deploy.sol"
+
+echo "Creating build artifacts..."
+forge build -o artifacts
+
+echo "Creating networks.json..."
+for deployment in "$deployment_output_folder/"*; do
+  # The subfolder name is the chain id
+  chain_id=${deployment##*/}
+  # First, every single deployment is formatted as if it had its own networks.json
+  jq --arg chainId "$chain_id" '
+    .transactions[]
+    | select(.transactionType == "CREATE" )
+    | {(.contractName): {($chainId): {address: .contractAddress, transactionHash: .hash }}}
+  '  <"$deployment/deployment.json"
+  # Then, all these single-contract single-chain-id networks.jsons are merged
```

Comment:
Should this comment go one line further down, as it is done after the for loop is done?

Reply:
Fixed! Adding comments to inline commands in Bash is a bit tricky, which is why I cheated my way out by writing it one line before.